Sources/SwiftDocCUtilities/ArgumentParsing/Options/PreviewOptions.swift

```diff
@@ -17,6 +17,13 @@ import Foundation
 public struct PreviewOptions: ParsableArguments {
     public init() { }
 
+    @Option(
+        name: .shortAndLong,
```

Comment:
I think this short name conflicts with the short name for command help: `-h, --help`. 

I'm not sure which of the two we would want to use `-h` for, if its better to have a short name for a dedicated option or it it's better to be consistent with the other commands.

Reply:
Good catch, thank you. Given that `-h` already means `--help`, I suppose we’d better leave it alone. I’ve switched `--host` to use a long option only.